Microsoft.Toolkit.Uwp.Services/Services/Bing/BingDataProvider.cs

```diff
@@ -41,7 +42,7 @@ public class BingDataProvider : DataProviderBase<BingSearchConfig, BingResult>
         protected override async Task<IEnumerable<TSchema>> GetDataAsync<TSchema>(BingSearchConfig config, int maxRecords, IParser<TSchema> parser)
         {
             var countryValue = config.Country.GetStringValue();
-            var locParameter = string.IsNullOrEmpty(countryValue) ? string.Empty : $"loc:{countryValue}+";
+            var locParameter = string.IsNullOrEmpty(countryValue) ? $"loc:{CultureInfo.CurrentCulture.Name.Split('-')[1].ToLower()}+" : $"loc:{countryValue}+";
```

Comment:
According to MSDN:

"The format for the culture name based on RFC 4646 is languagecode2>-country/regioncode2, where languagecode2 is the two-letter language code and country/regioncode2 is the two-letter subculture code. "

I looked up for RFC 4646 a bit and didn't see any conclusion or exclusion regarding this.